src/init.cpp

```diff
@@ -1619,7 +1619,8 @@ bool AppInitMain(InitInterfaces& interfaces)
             // first suggest a reindex
             if (!fReset) {
                 bool fRet = uiInterface.ThreadSafeQuestion(
-                    strLoadError + ".\n\n" + _("Do you want to rebuild the block database now?").translated,
+                    strprintf(_("%s.\n\nDo you want to rebuild the block database now?"), strLoadError),
```

Comment:
Not sure if we want to translate `%s.\n\n`, maybe add an `operator+` or `operator<<` to bilingual_str, so that either `+` or `strprintf` can be used.

Reply:
This line is outdated due to the following commits ;)

> ... maybe add an `operator+` or `operator<<` to bilingual_str, so that either `+` or `strprintf` can be used.

Agree. Good idea for the following PRs (as this PR does not touch `bilingual_str` itself).

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/16224#issuecomment-623798715).